pkg/autodiscovery/listeners/ecs.go

```diff
@@ -169,6 +173,9 @@ func (l *ECSListener) createService(c metadata.ContainerMetadata, firstRun bool)
 	image := c.Image
 	labels := c.Labels
 	svc.ADIdentifiers = ComputeContainerServiceIDs(svc.GetEntity(), image, labels)
+	if checkNames, err := getCheckNamesFromLabels(labels); err == nil {
+		svc.checkNames = checkNames
```

Comment:
in others services, in case or error, the error is logged with `log.Errorf()`. It can be good to have the same behavior between Services